doc/api/util.md

```diff
@@ -31,7 +31,7 @@ For example:
 const util = require('util');
 
 async function fn() {
-  return await Promise.resolve('hello world');
+  return Promise.resolve('hello world');
```

Comment:
Quoting https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function

> (...) because the return value of an async function is implicitly wrapped in Promise.resolve.

So I assume you can drop the `Promise.resolve` here and in below occurences

Reply:
@Florian-R Done. Thanks for the suggestion!